### PR TITLE
feat(ui): stream inspector

### PR DESCRIPTION
**Pull request by jmar910**
### Description
Adds stream inspector functionality to the UI

### Demo
Screen Shot 2022-12-06 at 9 31 27 AM
Screen Shot 2022-12-06 at 9 29 07 AM
Screen Shot 2022-12-06 at 9 29 16 AM

### Quick checks:

- [x] I have followed the [Code Guidelines](https://github.com/ConduitIO/conduit/blob/main/docs/code_guidelines.md).
- [x] There is no other [pull request](https://github.com/ConduitIO/conduit/pulls) for the same update/change.
- [x] I have written unit tests.
- [x] I have made sure that the PR is of reasonable size and can be easily reviewed.

### CODE REVIEW

File: ui/app/components/pipeline-editor/nodes/connector-node.js [hunk 1]
```diff
@@ -2,25 +2,35 @@ import Component from '@glimmer/component';
...
+  websockets;
+
+  registerNode(element, [nodeType, pipelineNodeManager, model, websockets]) {
     if (nodeType === 'source') {
       pipelineNodeManager.registerSourceNode(element, model);
     }
 
     if (nodeType === 'destination') {
       pipelineNodeManager.registerDestinationNode(element, model);
+      websockets.connect(
```
Comment (jayjayjpg) [inline ui/app/components/pipeline-editor/nodes/connector-node.js:18]: This is so cool! :sparkles:

File: ui/app/services/websockets.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
+import Service from '@ember/service';
+import { tracked } from '@glimmer/tracking';
+import { prettyPrintJson } from 'pretty-print-json';
+import { htmlSafe } from '@ember/template';
+import { run, schedule } from '@ember/runloop';
```
Comment (jayjayjpg) [inline ui/app/services/websockets.js:5]:
```suggestion
import { run } from '@ember/runloop';
```

File: ui/app/services/websockets.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+    this._socket.addEventListener('message', (event) => {
+      if (!this.isTrackingRecords) {
+        return;
+      }
+
+      const data = JSON.parse(event.data);
+
+      run(() => {
+        if (this._records.length >= 10) {
```
Comment (jayjayjpg) [inline ui/app/services/websockets.js:31]: Just curious: are records with a length of less than 10 characters usually just meta data or something else that's not really needed in the stream view?

**Review by jayjayjpg — COMMENTED** [review]
This already reads great and super excited to see this one come live! Happy to give this a test run tomorrow if it's valuable, let me know and awesome work! :sparkles:

File: ui/app/services/websockets.js (hunk 3 above)
Comment (jmar910) [reply ui/app/services/websockets.js]: Its not the length of the record, but the length of the records array. We only hold in memory and display the last 10 records from the websocket

Comment (jayjayjpg) [reply ui/app/services/websockets.js]: Got it and thank you for the outline!

**Review by jayjayjpg — APPROVED** [review]
This reads great and works well locally -- awesome work on this new :sparkles: feature!

**Pull request merged**